framework/contrib/csvdiff/csvdiff

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python2.7
```

Comment:
I don't think that this script should be in contrib. Contrib is for external projects that we've copied into MOOSE. If you want to open source this as a project under your profile, then it would be appropriate for it to be a contrib. As it stands it's a utility that's just part of MOOSE. I might recommend that you just put it in the scripts directory (moose/scripts).